### PR TITLE
Update RulesConfigYamlTest

### LINKED ISSUE #304: Update RulesConfigJsonTest
Now it checks that only one config contain rule. But it should checks all configuration files.

### PR DESCRIPTION
**Pull request by kentr0w**
### What's done:
    Added test for another config
   closes #304 
## Fixme
   Check third config file

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt [hunk 1]
```diff
@@ -7,28 +7,36 @@ import org.cqfn.diktat.common.config.rules.getRuleConfig
...
-class RulesConfigJsonTest {
+class RulesConfigYamlTest {
     @Test
-    fun `read rules config json`() {
-        val allRulesFromConfig = readAllRulesFromConfig()
+    fun `read rules config yml`() {
+        compareRulesAndConfig("diktat-analysis.yml")
+        compareRulesAndConfig("diktat-analysis-huawei.yml")
+        val thirdConfig = "${System.getProperty("user.dir")}${File.separator}..${File.separator}diktat-analysis.yml${File.separator}"
+        compareRulesAndConfig(thirdConfig, "diktat-analysis.yml")
```
Comment (aktsay6) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt:22]: It is hard to differ whether it is parent's(diktat) diktat-analysis.yml or diktat-rules

**Review by aktsay6 — APPROVED** [review]
lgtm

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt (hunk 1 above)
Comment (akuleshov7) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt]: yeah, better to print absolute path to a file here

**Pull request merged**